README.md

```diff
@@ -1,10 +1,113 @@
+
+> [!Note] 
+> Only frames that will be extracted for SAM2 processing should be annotated with positive / negative clicks or entries/exits. These frames will display a message "SAM2 Frame: Annotate Fish Position". 
+> Bite clicks can be annotated on any frame. 
+
+Annotations can be deleted using the Delete Selected or Delete All buttons. The Delete All button will prompt for confirmation before deleting all annotations.
+
+### Entries and Exits
+
+On the first SAM2 frame where the individual is visible, an Entry should be marked using the Add Entry button or the keystroke 'e'. On the last SAM2 frame where the individual is visible, an Exit should be marked using the Add Exit button or the keystroke 'x'. No location is associated with entries and exits, but the correct fish family and individual name should be specified before marking entries and exits. 
+Multiple entries and exits can be associated for each object; these will essentially tell SAM2 when to start and stop tracking the object. If the object leaves the field of view temporarily, it is best to mark it as an exit and re-entry; otherwise, SAM2 will keep looking for that object and may incorrectly mask a different but similar-looking object.
+
+If your cursor is in one of the input text boxes, pressing "e" or "x" will simply type in the text box and not create a new entry or exit. Click on the annotation table to remove your cursor from the input text box and use the hotkey shortcuts.
+
+### Saving Annotations
+
+To save your annotations, write the file name you wish to save as in the provided input textbox. If this file name has already been used, it will overwrite any existing files. 
+By default, saving your annotations will create two files that will be saved in the working directory where you launched the GUI from. One of these will be `filename_bites.csv`, and this will contain your bites annotations. Another file named `filename_annotations.npy` will contain your positive and negative positional annotations, entries, and exits.  The `filename_annotations.npy` file is what you will submit to SAM2 to track your individuals.
+Use the checkboxes at the top of the GUI to specify which files you'd like to save before clicking the Save Annotations button.
+
+To edit previous annotations or continue previous progress, use the Import Previous Annotations button to re-load your bites and locations annotations into the GUI.
+
+When all individuals to be tracked in a video have been marked with an entry, an exit, at least one positive click, you can save your annotations, exit the GUI, and proceed to SAM2 frame extraction for processing.
+
+## Extract Frames for SAM2
+
+The frames from a video will need to be extracted and stored within a folder before SAM2 processing:
```

Comment:
```suggestion
The frames from a video will need to be extracted and stored within a folder before SAM2 processing. This can be done from the command line using FFmpeg:
```